### PR TITLE
chore: 文案修正

### PR DESCRIPTION
**Pull request by xingwanying**
### 👀 PR includes

🔧 Chore

- [ ] Test case
- [x] Docs / demos update
- [ ] CI / workflow
- [ ] Release version
- [ ] Other ()

### CODE REVIEW

File: packages/s2-core/src/cell/data-cell.ts [hunk 1]
```diff
@@ -174,7 +174,7 @@ export class DataCell extends BaseCell<ViewMeta> {
     this.drawInteractiveBgShape();
     this.drawConditionIntervalShape();
     this.drawInteractiveBorderShape();
-    this.drawTextShape();
+    // this.drawTextShape();
```
Comment (lijinke666) [inline packages/s2-core/src/cell/data-cell.ts:177]: 手滑?

Comment (lijinke666) [reply packages/s2-core/src/cell/data-cell.ts:177]: 终于体现一次单测的作用了 
image

Comment (xingwanying) [reply packages/s2-core/src/cell/data-cell.ts:177]: 雀食

**Review by wjgogogo — APPROVED** [review]

**Pull request merged**